backend/substrapp/views/filters_utils.py

```diff
@@ -71,6 +71,13 @@ def _same_nature(filter_key, object_type):
     return {filter_key, object_type} <= {'algo', 'composite_algo'}
 
 
+def _model_tuple(model):
+    if 'compositeTraintuple' in model:
+        return model['compositeTraintuple']
+
+    return model['traintuple']
+
+
```

Comment:
can be replace by `model.get('compositeTraintuple', model['traintuple'])` no ?

Reply:
So it will be reduce to `x.get('compositeTraintuple', x['traintuple'])`

Reply:
We'll also get aggregateTuples very soon, so that would become `model.get('aggregateTuple', model.get('compositeTraintuple', model['traintuple'])`. If that's still fine with you I'll update the code with your suggestion, otherwise I'll leave it as it is.

Reply:
So maybe it better to do 
``` python
def _model_tuple(model):
    if 'compositeTraintuple' in model:
        return model['compositeTraintuple']
    elif 'aggregateTuple' in model
        return model['compositeTraintuple']
    elif 'traintuple' in model:
        return model['traintuple']
    else:
        raise NotImplementedError

```
Or something like that

Reply:
👍